src/main/java/hudson/plugins/git/extensions/impl/PathRestriction.java

```diff
@@ -127,8 +127,12 @@ public Boolean isRevExcluded(GitSCM scm, GitClient git, GitChangeSet commit, Tas
             }
         }
 
-        // If every affected path is excluded, return true.
-        if (includedPaths.size() == excludedPaths.size()) {
+       if (excluded.isEmpty() && !included.isEmpty() && includedPaths.isEmpty()) {
```

Comment:
Sorry my laptop was broken and seems i started missing many actions from different PRs. I remember that i created msg exclusion extension, so decided to look on PR when got email :)